addons/website_event/models/event_event.py

```diff
@@ -640,17 +642,38 @@ def search_in_address(env, search_term):
             'search_tags': search_tags,
             'no_date_domain': no_date_domain,
             'no_country_domain': no_country_domain,
+            'template_key' : 'website_event.search_items_event',
+            'group_title' : self.env._('Events'),
         }
 
+    def _get_lowest_ticket_price(self):
+        self.ensure_one()
+        # Check if the price field exists on tickets
+        price_exists = "price" in self.env['event.event.ticket']._fields
+        # If no price field exists at all, skip everything
+        if not price_exists:
+            return False
+
+        lowest_price = None
```

Comment:
```suggestion
        lowest_price = float('inf')
```
Assign max number & then compare for lowest.